Assignment_Week_3.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace MICP_Assignments
+{
+    public class Assignment_Week_3
+    {
+        public static int[][] uniqueTripletCombinations(int[] userInput)
+        {
+            //checks to ensure that there IS input into the array
+            if (userInput == null || userInput.Length <= 0)
+                return null;
+
+            //if there isn't enough input
+            if (userInput.Length < 3)
+                throw new ArgumentException();
+
+            //used to contain the pending results
+            List<int[]> results = new List<int[]>();
+
+            //organizes the user input
+            Array.Sort(userInput);
+

```

Comment:
Hey! maybe here you don't need to sort the array because it doesn't impact your solution.